server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/CVRToAuditDownload.java

```diff
@@ -234,10 +235,36 @@ public String endpointBody(final Request the_request, final Response the_respons
             ComparisonAuditController.computeBallotOrder(cdb, index, ballot_count, 
                                                          duplicates, audited);
       }
-     
+
       response_list.addAll(BallotSelection.toResponseList(cvr_to_audit_list));
-      response_list.sort(new BallotOrderComparator());
-      
+      response_list.sort(null);
+
+      // Add audit board information if the round is present
+      if (round.isPresent()) {
+        final Round roundObject = cdb.rounds().get(round.getAsInt() - 1);
+
+        final List<Map<String, Integer>> bsa =
+          roundObject.ballotSequenceAssignment();
+
+        if (bsa != null) {
+          // Walk the sequence assignments getting the audit boards' index and
+          // count values. Use that information to set the audit board index for
+          // each response row.
+          for (int i = 0; i < bsa.size(); i++) {
+            final Map<String, Integer> m = bsa.get(i);
+
+            final Integer boardIndex = m.get("index");
+            final Integer boardCount = m.get("count");
+
+            for (int j = boardIndex; j < boardIndex + boardCount; j++) {
+              // TODO: Will this always agree with the round information?
+              final CVRToAuditResponse row = response_list.get(j);
```

Comment:
:spiral_notepad: Tentatively the answer is "yes", but deserves scrutiny.

Reply:
:thinking: